festim/mesh/mesh_1d.py

```diff
@@ -61,3 +62,49 @@ def check_borders(self, volume_subdomains):
             or sorted_borders[-1] != self.vertices[-1]
         ):
             raise ValueError("borders dont match domain borders")
+
+    def define_meshtags(self, subdomains):
```

Comment:
It would simplify the method if we have instead of `subdomains`, `surface_subdomains` and `volume_subdomains`.

```python
        for sub_dom in subdomains:
            if isinstance(sub_dom, F.SurfaceSubdomain1D):
```
would become

```python
        for surf in surface_subdomains:
            ...
        for vol in solume_subdomains:
            ...
```